### ISSUE TITLE
False positive for clipboard.js: CVE-2014-5509

### ISSUE THREAD

**Issue by exortech** [anchore/grype#728]
**What happened**:

CVE-2014-5509 (https://nvd.nist.gov/vuln/detail/CVE-2014-5509), which is a Perl utility, is being associated with clipboard.js (https://github.com/zenorocha/clipboard.js) - presumably because they share a common name and have overlapping versions.

**What you expected to happen**:

This CVE should not be associated with the clipboard.js module.

**How to reproduce it (as minimally and precisely as possible)**:

> npm init
> npm install clipboard --save
> syft . -o json > clipboard-sbom.json
> grype sbom:./clipboard-sbom.json

**Anything else we need to know?**:

**Environment**:
- Output of `grype version`: 
> Application:          grype
> Version:              0.35.0
> Syft Version:         v0.44.0
> BuildDate:            2022-04-13T21:41:54Z
> GitCommit:            4ed0704dcf708124272caf26de8aec916c04a135
> GitDescription:       v0.35.0
> Platform:             darwin/arm64
> GoVersion:            go1.18
> Compiler:             gc
> Supported DB Schema:  3

**Comment by exortech** [comment]
I assume that this could have something to do with the warnings in the output from grype:
> [0000]  WARN unknown package metadata type="" for packageID="b46d59685f226263" from-lib=syft

Perhaps grype does not recognize type="npm"?

**Comment by exortech** [comment]
Here's the associated output from the SBOM file:
```
  {
   "id": "b46d59685f226263",
   "name": "clipboard",
   "version": "2.0.10",
   "type": "npm",
   "foundBy": "javascript-lock-cataloger",
   "locations": [
    {
     "path": "package-lock.json"
    }
   ],
   "licenses": [],
   "language": "javascript",
   "cpes": [
    "cpe:2.3:a:clipboard:clipboard:2.0.10:*:*:*:*:*:*:*",
    "cpe:2.3:a:*:clipboard:2.0.10:*:*:*:*:*:*:*"
   ],
   "purl": "pkg:npm/clipboard@2.0.10"
  },

```

**Comment by willmurphyscode** [comment]
Hi @exortech, thanks for reporting this issue!

I'm no longer able to reproduce this false positive:

```
mkdir clipboard && cd clipboard && npm init -y && npm install clipboard --save && grype dir:.
```

prints `No vulnerabilities found`.

I believe this was fixed by https://github.com/anchore/grype/releases/tag/v0.60.0, where grype's matching behavior for npm packages disables CPE matching by default now. Please let us know if you find this is still happening.

**Comment by exortech** [comment]
Thanks for reviewing! All's good with closing the issue.

**Comment by willmurphyscode** [comment]
Added `changelog-ignore` because this was fixed in `0.60.0` and so shouldn't be included in the current release's release notes.
